src/views/docs/en/reference/runtime-helpers/node.js.md

```diff
@@ -101,19 +105,23 @@ arc.events.publish({
 
 ---
 
-### `arc.http`
+## `arc.http`
 
-`arc.http` provides middleware and request/response normalization for `@http` functions using Express-style callbacks.
+`arc.http` provides middleware and request/response normalization for `@http` functions using your choice of `async` functions or Express-style callbacks. Declare HTTP routes with the [`@http`](/docs/en/reference/project-manifest/http) pragma.
 
-#### Requests
+> A legacy `arc.http.async` middleware interface will remain available exclusively for `async` functions, although we strongly encourage using the unified `arc.http` interface.
 
-`arc.http` (and `arc.http.async`) provide the following:
-- Support for request formats from both AWS API Gateway `HTTP` and `REST` APIs
-- Backward-compatibility for `REST` API properties in `HTTP` APIs, enabling seamless API upgrades
+
+### Requests
+
+`arc.http` provides the following:
+- Built-in session support
 - Added conveniences, such as automatic parsing of `req.body`
+- Support for request formats from both AWS API Gateway `HTTP` and `REST` APIs
 - Added properties commonplace in other web servers, such as `req.params` (as opposed to the much more verbose `req.queryStringParameters`)
```

Comment:
I realize this line wasn't changed in this patch, but I think it should be

> such as `req.query` [...]

`req.params` is an alias for `pathParameters`